metrics/parse_tfm.py

```diff
@@ -64,11 +64,14 @@ def __init__(self, start_char, end_char, char_info, width_table,
         self.ligkern_program = LigKernProgram(ligkern_table)
         self.kern_table = kern_table
 
-    def get_char_metrics(self, char_num):
+    def get_char_metrics(self, char_num, zero_indexed=False):
```

Comment:
There was one font `rsfs` where glyph indices were messed up.  This corrects them.  I'll add a docstring.